source/tutorial/triggers-fts.txt

```diff
@@ -0,0 +1,345 @@
+      </atlas-ui/documents/#view-documents>` in the ``todo.alerts`` collection
+      to confirm that the terms appear in Atlas.
+
+Add a Trigger
+-------------
+
+After fully configuring the alert box, create an :ref:`Atlas trigger <triggers>` 
+that alerts you when a new task contains one of your alert terms. Triggers can execute 
+application and database logic in response to a change event. Each trigger links to an 
+:ref:`Atlas Function <functions>` that defines the trigger's behavior.
+
+In this section, you create a :ref:`database trigger <database-trigger>` 
+that runs whenever a user creates a new task. In the trigger's 
+function, you define:
+
+- The message that displays in your application logs.
+
+- The database logic, so that the trigger returns the message only when the 
+  document contains an alert term.
+
+- An Atlas Search query that aggregates other tasks that contain 
+  the same alert term.
+
+.. procedure::
+
+   .. step:: Create a Search Index
+
+      In order to run Atlas Search queries on your data, you must first create
+      an :atlas:`Atlas Search index </atlas-search/create-index/>` to map the fields
+      in your collection.
+
+      Create a search index on the ``todo.Item`` collection by following the procedure 
+      described in :atlas:`Create an Atlas Search Index </atlas-search/create-index/#procedure>`.
+      When creating your index, leave the :guilabel:`Index Name` as ``default`` and proceed
+      with the default index configuration settings.
+   
+   .. step:: Add a Database Trigger
+   
+      To open the database trigger configuration page in the App Services UI:
+
+      A. Click the :guilabel:`App Services` tab and select the tile for your 
+         app.
+
+      #. In the left navigation menu, click :guilabel:`Triggers`.
+      
+      #. Click :guilabel:`Add a Trigger` and leave the :guilabel:`Trigger type`
+         set to :guilabel:`Database`. 
+         
+      #. Name the trigger ``sendAlerts``.
+
+      #. Configure the trigger to listen only for new task documents 
+         inserted into the ``todo.Item`` collection:
+
+         - For the :guilabel:`Cluster Name`, select 
+           the deployment that's synced to the template app. 
+
+         - For the :guilabel:`Database Name` and :guilabel:`Collection Name`, 
+           select the ``todo`` database and ``Item`` collection.
+
+         - For the :guilabel:`Operation Type`, select :guilabel:`Insert`.
+
+         - Enable :guilabel:`Full Document` to include each new report document
+           in the change event passed to the trigger function.
+
+   .. step:: Define an Atlas function
+
+      To define the trigger's function, navigate to the :guilabel:`Function` 
+      section of the configuration page and select :guilabel:`+ New Function` 
+      from the drop-down menu. Name the function ``triggers/sendAlerts``.
+      
+      The following JavaScript function returns a message in your application
```

Comment:
I read "the following JS function" but then had to actually scroll the page down past some text to see the code. The code block also has an intro sentence that uses "the following" too. Can we rework this section so that "the following" only comes immediately before the code block?

One approach - we can move the code block up and then have all the explanation below it. This would also help to highlight the currently orphaned "click save and deploy" instruction at the end of the step.

e.g.

1. Name the function `triggers/sendAlerts`
2. Copy the following code into the function body
3. Here's what it does....
4. Click save and deploy